ci/build_container/build_container.sh

```diff
@@ -37,8 +37,18 @@ fi
 
 export THIRDPARTY_DEPS=/tmp
 export THIRDPARTY_SRC=/thirdparty
-export THIRDPARTY_BUILD=/thirdparty_build
```

Comment:
This directory can be deleted now, right? I think this should get added into the script, or at least into the PR description that bazel clean is required.